### DISCUSSION TITLE
Multiple Base files

### DISCUSSION THREAD

**Discussion by EugeneKonneh** [mjhea0/cypress-visual-regression#257] [General]
I am new to regression testing.
I have used this cypress to create a base image for a web element.

How do I create multiple base images, each of a different page such that I can test successive regression tests as I navigate through the AUT?

Thank you for your assistance.
Eugene

**Comment by Zaista** [comment]
Hi, first create a new test by calling the `cy.compareSnapshot` command with different names at different points in your test, for example:
```
it('some test', () => {
  cy.visit('some/url')
  cy.compareSnapshot('home-page')
  cy.get('#another-page-link').click()
  cy.compareSnapshot('another-page')

  // and so on
})
```
Then set the environment variable `visualRegressionType` to 'base' and run it locally once.
This will generate two base images: `home-page` and `another-page` image

Then change the env variable to 'regression', commit and push everything (with base images) to source control, then schedule your tests to run.

Hope this helps, happy testing
